docs/features/http.md

```diff
@@ -0,0 +1,102 @@
+# HTTP Step
+
+Kahpp makes it possible to reach one or more HTTP Endpoints.  
+
+### Usage
+
+First, we need to declare the API on the Kahpp instance to do that.
+
+For example:
+```yaml
+kahpp:
+  apis:
+    my-dummy-api:
+      basePath: http://my-dummy-api
+      options: # Optional
+        rateLimit:
+          requestsPerSecond: 20
+          warmUpMillis: 2000
+        connection:
+          connectTimeoutMillis: 300
+          socketTimeoutMs: 1500
+        retries:
+          retryOnTimeout: true
+          statusCodeRetryTimeSeedInMs: 500
+          statusCodeRetryTimeCapInMs: 5000
+          statusCodes:
+            - statusCodeStart: 500
+              statusCodeInclusiveEnd: 599
+              retries: 10
+```
+Then in the steps, we may decide how to use it and how to handle the response.
+
+## Ok or Produce Error
+In case of error, it routes the message to a specific topic.
+
+Simple Http Call
```

Comment:
Some places in this file use `Http` others `HTTP`. Which one should we stick with?

Reply:
HTTP